  * # <font color='blue' size='5'>Google will shutoff GoogleCode on Jan.1, 2014. New version will be hosted on <a href='https://sourceforge.net/projects/rseqc/'>Sourceforge</a></font> #


  * # <font color='blue' size='5'> <a href='http://rseqc.sourceforge.net/'>Online documentation</a> </font> #

  * # <font color='blue' size='5'>Output Examples</font> #
![http://dldcc-web.brc.bcm.edu/lilab/liguow/RSeQC/figure/Supplementary_Figure1.png](http://dldcc-web.brc.bcm.edu/lilab/liguow/RSeQC/figure/Supplementary_Figure1.png)
(A) Raw sequence quality visualized by box plot. All reads are overlaid together, and then summarize Phred quality score for each position of read (or sequencing cycle).  (B) Raw sequence quality visualized by heatmap. All reads are overlaid together, and then calculate frequency of nucleotide with certain Phred quality score for each position of read. Color indicates nucleotide density."blue" low density, "orange" median density, "red" high density. (C) Nucleotide composition bias visualized by NVC (Nucleotide frequency versus sequencing cycle) plot. All reads are overlaid together, and then calculate nucleotide frequency for each position of read (or sequencing cycle). (D) Reads duplication rate. "Sequence-base", reads with exactly the same sequence are  regarded as duplicates; "Mapping-base" reads mapped to the same location are regarded as duplicates.(E) Raw sequence GC content histogram. (F) Coverage uniformity over gene body. (G) Mapped reads distribution over genome structure. tss= transcription  start site, tes=transcription end site, ds=downstream, us=upstream. (H) Insert size histogram with fitted curve.(I) clipping profile of mapped reads. (J) RPKM saturation of 25% highest expressed genes (see Manual). (K)RPKM saturation for a particular transcript (see Manual). (L) Splice junction annotation. "known", both splice sites (3'SS and 5'SS) can be annotated by reference gene model; "partial novel" only one splice site can be annotated; "complete novel" neither of the splice site can be annotated.


# <font color='blue'>Reference</font> #
Wang, L., Wang, S., & Li, W. (2012). RSeQC: Quality Control of RNA-seq experiments. [Bioinformatics](http://bioinformatics.oxfordjournals.org/content/28/16/2184.long) (Oxford, England). doi:10.1093/bioinformatics/bts356

# <font color='blue'>Contact</font> #
  * [RSeQC google group:](http://groups.google.com/group/rseqc-discuss) rseqc-discuss@googlegroups.com


<img src='https://sites.google.com/site/liguowangspublicsite/home/images.jpg' width='85' height='85'>
<img src='https://sites.google.com/site/liguowangspublicsite/home/BCM_Logo_RGB.jpg' width='120' height='60'>
<img src='https://sites.google.com/site/liguowangspublicsite/home/southeast.jpg' width='70' height='70'>

